osu.Game/Notifications/NotificationIcon.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Graphics.Colour;
+using osu.Game.Graphics;
+
+namespace osu.Game.Notifications
+{
+    public class NotificationIcon
+    {
+
+        /// <summary>
+        /// The icon displayed in the notification.
+        /// </summary>
+        public FontAwesome Icon { get; }
+
+        /// <summary>
+        /// Changes to background color of the icon. Defaults to grey
+        /// </summary>
+        public ColourInfo BackgroundColour { get; }

```

Comment:
These properties don't have bindings like all the others inside `Notification`.

In general, I'm not too certain about the bindings, versus just having them as fields and the notifications updating inside their `Update()`.

Reply:
this is intentional. if you want to make changes you'd replace the `NotificationIcon` itself.

Reply:
That isn't enforced anywhere, considering these properties + NotificationIcon are both public.

Nothing stops me from wanting to alter just the background colour and thinking i can do `NotificationIcon.BackgroundColour = ....`

Reply:
In my working through this PR in discord, i told him to make these `readonly`. It does need to be enforced.

Reply:
Get only properties are readonly, aren't they?